.env.example

```
@@ -0,0 +1,17 @@
+SERVER_PORT=
+
+DB_PASSWORD=
+POSTGRES_HOST=
+POSTGRES_PORT=
+POSTGRES_DB=
+POSTGRES_USER=
+
+KEYCLOAK_CLIENT_SECRET=
+
+EUREKA_SERVER_URL=
+CONFIG_SERVER_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONFIG_SERVER_URL key should go before the EUREKA_SERVER_URL key

(UnorderedKey)